styles/bsky/catppuccin.user.css

```diff
@@ -64,16 +64,19 @@
     }
 
     /* secondary text */
-    [style*="rgb(185, 185, 193)"] {
+    [style*="rgb(163, 178, 194)"] {
       color: @subtext1 !important;
     }
 
     /* generic blue inline */
-    [style*="color: rgb(0, 133, 255)"] {
+    [style*="color: rgb(16, 131, 254)"] {
       color: @accent-color !important;
     }
 
     /* blue svg inline */
+    [fill="hsl(211, 99%, 53%)"] {
+      fill: @accent-color;
+    }
     [fill="#0085ff"] {
       fill: @accent-color;
     }
```

Comment:
```suggestion
    /* blue svg inline */
    [fill="hsl(211, 99%, 53%)"],
    [fill="#0085ff"] {
      fill: @accent-color;
    }
```

Reply:
oh yea, i didn't have the compact syntax in my original theme and was a bit too tired to be bothered to refactor. I'll do the changes you mentioned when I get home later today; as commit signing on my iPad is currently broken (?)